docs/modules/ROOT/pages/python-client-with-pandas-and-DBAPI.adoc

```diff
@@ -0,0 +1,303 @@
+plt.show()
+client.shutdown()
+----
+
+In this example, we have created a map named `patients` and stored the blood pressure and heart rate data of fifty patients in a DataFrame and load it to Hazelcast cluster as JSON.
+Then, we have retrieved the data of a random patient and plotted it.
+
+== Using DPABI API with Pandas
+Hazelcast have a https://hazelcast.readthedocs.io/en/stable/api/db.html[DBAPI] which is compatible with https://pandas.pydata.org/docs/reference/index.html#api[Pandas]. You can use this API to analyze your data in Hazelcast cluster with Pandas.
+For DBAPI, we are going to use another example. So, do not forget that these two are completely different examples.
+Let's create a map, we have given a different name to the map name send to cluster, since if you are using Hazelcast Viridian as cluster
+we do not want to override the data to the map that we have created in the previous example.
+
+[source, python]
+----
+import hazelcast
+from hazelcast.db import connect
+from hazelcast.core import HazelcastJsonValue
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+
+client = hazelcast.HazelcastClient()
+
+# Create a Distributed Map in the cluster
+patients = client.get_map("patientsWithDBAPI").blocking()
+----
+
+Now, we are going to create Connection object instance and a cursor object instance. We will use these instances to execute our SQL queries.
+
+[source, python]
+----
+# Create a connection object
+conn = connect()
+cursor = conn.cursor()
+----
+
+Now, we are ready to execute our queries. Let's create a table named `patients` first.
+
+[source, python]
+----
+# Create a table named patients
+q = """
+CREATE MAPPING patients (
+    __key INT,
+    this JSON
+)
+TYPE IMaP
+OPTIONS (
+    'keyFormat' = 'int'
+    'valueFormat' = 'json'
+)
+"""
+cursor.execute(q)
+----
+
+Now, we are going to insert our data to the table. We are going to use Pandas to create our data.
+
+[source, python]
+----
+for pid in range(0, 50):
+    # Create DataFrame with random values
+    df = pd.DataFrame(
+        data={
+            "blood_pressure": np.random.randint(80, 120, size=(75,)),
+            "heart_rate": np.random.randint(60, 100, size=(75,)),
+        },
+        index=pd.date_range("2023-01-15", periods=75, freq="H"),
+    )
+    # Load DataFrame to Hazelcast cluster as HazelcastJsonValue
+    cursor.execute(f'INSERT INTO "{patients}" VALUES (?, ?)', (pid, HazelcastJsonValue(df.to_json())))
+----
+
+We have our data in cluster now. Let's retrieve the data of a random patient and plot it.
+
+[source, python]
+----
+import hazelcast
+from hazelcast.db import connect
+from hazelcast.core import HazelcastJsonValue
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+
+client = hazelcast.HazelcastClient()
+
+# Create a Distributed Map in the cluster
+patients = client.get_map("patientsWithDBAPI").blocking()
+
+conn = connect()
```

Comment:
this will create a separate client, why do we have the `client = hazelcast.HazelcastClient()` in this example